kenjutsu/blocks.py

```diff
@@ -12,7 +12,7 @@
 import kenjutsu.format
 
 
-def split_blocks(space_shape, block_shape, block_halo=None):
+def split_blocks(space_shape, block_shape, block_halo=None, index=False):
```

Comment:
Though we are adding this flag, would like to see it already be deprecated. Meaning we should cycle through once having this set to `True` by default and then drop the flag altogether.